tests/test_config_parsing/test_res_config.py

```diff
@@ -17,8 +17,8 @@ def touch(filename):
         fh.write(" ")
 
 
-@pytest.mark.skip(reason="https://github.com/equinor/ert/issues/2554")
-def test_res_config_simple_config_parsing(tmpdir):
+@pytest.mark.skip(reason="Should pass after #4178")
```

Comment:
is there a note in #4178 stating part of that job is to "un-skip" this test and make it pass?

Reply:
yeah Morten put a remark there : )

Reply:
I think perhaps it would be a good idea to just keep the issue number here. Also, we could use xfail now since it does not abort any longer.

Reply:
The original issue was not an issue any longer. But the new issue has now been put in a xfail.